go/enclave/enclave_admin_service.go

```diff
@@ -74,6 +76,21 @@ func NewEnclaveAdminService(config *enclaveconfig.EnclaveConfig, logger gethlog.
 	}
 }
 
+func (e *enclaveAdminService) MakeActive() common.SystemError {
+	e.mainMutex.Lock()
+	defer e.mainMutex.Unlock()
+
+	if !e.isBackupSequencer() {
+		return fmt.Errorf("only backup sequencer can become active")
+	}
+	// todo
+	// change the node type service
+	// do something with the mempool
+	// make some other checks?
```

Comment:
Once we've got the sequencer Enclave IDs permission list monitoring we should include that check here probably. We could even make it so that all sequencer enclaves start as backup and it can't be activated until the permissioning is done?

Reply:
yes. good idea. I'll add this as a comment.